# virt-manager graphical console not displayed when opening the details display
# https://bugs.launchpad.net/getdeb.net/+bug/1355289
Index: virt-manager-1.1.0/virtManager/console.py
===================================================================
--- virt-manager-1.1.0.orig/virtManager/console.py	2014-09-09 23:19:18.396554149 +0200
+++ virt-manager-1.1.0/virtManager/console.py	2014-09-09 23:19:18.392554149 +0200
@@ -1406,10 +1406,10 @@
                 vmmConsolePages.connected_viewers -= 1
                 if vmmConsolePages.connected_viewers == 0:
                     if self.gtk_settings_mnemonic is not None:
-                        gtk.settings_get_default().set_property("gtk-enable-mnemonics", self.gtk_settings_mnemonic)
+                        Gtk.Settings.get_default().set_property("gtk-enable-mnemonics", self.gtk_settings_mnemonic)
             elif not self._viewer_connected and val:
                 if vmmConsolePages.connected_viewers == 0:
-                    settings = gtk.settings_get_default()
+                    settings = Gtk.Settings.get_default()
                     if has_property(settings, "gtk-enable-mnemonics"):
                         self.gtk_settings_mnemonic = settings.get_property("gtk-enable-mnemonics")
                         settings.set_property("gtk-enable-mnemonics", False)
